packages/web3-eth-iban/test/unit/iban.test.ts

```diff
@@ -15,7 +15,7 @@ You should have received a copy of the GNU Lesser General Public License
 along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-import { Iban } from '../../src/index';
+import { Web3Iban } from '../../src/web3_iban';
```

Comment:
All changes in this file should be reverted as its more good to use `Iban` name for making it more compatible to 1.x